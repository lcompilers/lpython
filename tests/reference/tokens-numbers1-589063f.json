{
    "basename": "tokens-numbers1-589063f",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/numbers1.py",
    "infile_hash": "396f6d7a3173c2ee59dd02b50f4b3092086d84e5d0ba01c88c8dd0dd",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "tokens-numbers1-589063f.stdout",
    "stdout_hash": "8d748858636ad37f74b820be328ef461de3d7d6446c4c67703f32bde",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}