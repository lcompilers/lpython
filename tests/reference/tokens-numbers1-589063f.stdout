(TOKEN "identifier" i) 0:0
(TOKEN "=") 2:2
(TOKEN "integer" 123) 4:6
(NEWLINE) 7:7
(TOKEN "identifier" i) 8:8
(TOKEN "=") 10:10
(TOKEN "-") 12:12
(TOKEN "integer" 123) 13:15
(NEWLINE) 16:16
(TOKEN "identifier" i) 17:17
(TOKEN "=") 19:19
(TOKEN "-") 21:21
(TOKEN "integer" 291) 22:26
(NEWLINE) 27:27
(TOKEN "identifier" i) 28:28
(TOKEN "=") 30:30
(TOKEN "integer" 6844) 32:37
(NEWLINE) 38:38
(TOKEN "identifier" i) 39:39
(TOKEN "=") 41:41
(TOKEN "integer" 6844) 43:48
(NEWLINE) 49:49
(TOKEN "identifier" i) 50:50
(TOKEN "=") 52:52
(TOKEN "-") 54:54
(TOKEN "integer" 83) 55:59
(NEWLINE) 60:60
(TOKEN "identifier" i) 61:61
(TOKEN "=") 63:63
(TOKEN "integer" 87) 65:70
(NEWLINE) 71:71
(TOKEN "identifier" i) 72:72
(TOKEN "=") 74:74
(TOKEN "-") 76:76
(TOKEN "integer" 13) 77:82
(NEWLINE) 83:83
(TOKEN "identifier" i) 84:84
(TOKEN "=") 86:86
(TOKEN "integer" 13) 88:93
(NEWLINE) 94:94
(TOKEN "identifier" i) 95:95
(TOKEN "=") 97:97
(TOKEN "real" 123.000000) 99:102
(NEWLINE) 103:103
(TOKEN "identifier" i) 104:104
(TOKEN "=") 106:106
(TOKEN "real" 0.123000) 108:111
(NEWLINE) 112:112
(TOKEN "identifier" i) 113:113
(TOKEN "=") 115:115
(TOKEN "real" 123.450000) 117:122
(NEWLINE) 123:123
(TOKEN "identifier" i) 124:124
(TOKEN "=") 126:126
(TOKEN "real" 0.123000) 128:132
(NEWLINE) 133:133
(TOKEN "identifier" i) 134:134
(TOKEN "=") 136:136
(TOKEN "real" 120000000000.000000) 138:144
(NEWLINE) 145:145
(TOKEN "identifier" i) 146:146
(TOKEN "=") 148:148
(TOKEN "real" 123400000000.000000) 150:158
(NEWLINE) 159:159
(TOKEN "identifier" i) 160:160
(TOKEN "=") 162:162
(TOKEN "real" 0.000000) 164:170
(NEWLINE) 171:171
(TOKEN "identifier" i) 172:172
(TOKEN "=") 174:174
(TOKEN "real" 0.000000) 176:183
(NEWLINE) 184:184
(TOKEN "identifier" i) 185:185
(TOKEN "=") 187:187
(TOKEN "real" 123400000000.000000) 189:196
(NEWLINE) 197:197
(TOKEN "identifier" i) 198:198
(TOKEN "=") 200:200
(TOKEN "integer" 12) 202:203
(TOKEN "+") 204:204
(TOKEN "imag number" 3.000000j) 205:206
(NEWLINE) 207:207
(TOKEN "identifier" i) 208:208
(TOKEN "=") 210:210
(TOKEN "integer" 12) 212:213
(TOKEN "+") 214:214
(TOKEN "imag number" 3.000000j) 215:216
(NEWLINE) 217:217
(TOKEN "identifier" i) 218:218
(TOKEN "=") 220:220
(TOKEN "integer" 12) 222:223
(TOKEN "+") 224:224
(TOKEN "imag number" 0.000000j) 225:233
(NEWLINE) 234:234
(TOKEN "identifier" i) 235:235
(TOKEN "=") 237:237
(TOKEN "real" 0.120000) 239:241
(TOKEN "+") 242:242
(TOKEN "imag number" 0.001000j) 243:247
(NEWLINE) 248:248
(NEWLINE) 249:249
(TOKEN "identifier" i) 250:250
(TOKEN "=") 252:252
(TOKEN "integer" 1234) 254:258
(NEWLINE) 259:259
(TOKEN "identifier" i) 260:260
(TOKEN "=") 262:262
(TOKEN "-") 264:264
(TOKEN "integer" 1234) 265:269
(NEWLINE) 270:270
(TOKEN "identifier" i) 271:271
(TOKEN "=") 273:273
(TOKEN "-") 275:275
(TOKEN "integer" 291) 276:281
(NEWLINE) 282:282
(TOKEN "identifier" i) 283:283
(TOKEN "=") 285:285
(TOKEN "integer" 6844) 287:293
(NEWLINE) 294:294
(TOKEN "identifier" i) 295:295
(TOKEN "=") 297:297
(TOKEN "integer" 4796) 299:305
(NEWLINE) 306:306
(TOKEN "identifier" i) 307:307
(TOKEN "=") 309:309
(TOKEN "-") 311:311
(TOKEN "integer" 83) 312:317
(NEWLINE) 318:318
(TOKEN "identifier" i) 319:319
(TOKEN "=") 321:321
(TOKEN "integer" 87) 323:329
(NEWLINE) 330:330
(TOKEN "identifier" i) 331:331
(TOKEN "=") 333:333
(TOKEN "-") 335:335
(TOKEN "integer" 43) 336:345
(NEWLINE) 346:346
(TOKEN "identifier" i) 347:347
(TOKEN "=") 349:349
(TOKEN "integer" 13) 351:358
(NEWLINE) 359:359
(TOKEN "identifier" i) 360:360
(TOKEN "=") 362:362
(TOKEN "real" 1123.000000) 364:369
(NEWLINE) 370:370
(TOKEN "identifier" i) 371:371
(TOKEN "=") 373:373
(TOKEN "real" 0.123000) 375:380
(NEWLINE) 381:381
(TOKEN "identifier" i) 382:382
(TOKEN "=") 384:384
(TOKEN "real" 0.123000) 386:392
(NEWLINE) 393:393
(TOKEN "identifier" i) 394:394
(TOKEN "=") 396:396
(TOKEN "real" 120000000000.000000) 398:406
(NEWLINE) 407:407
(TOKEN "identifier" i) 408:408
(TOKEN "=") 410:410
(TOKEN "real" 0.000001) 412:428
(NEWLINE) 429:429
(TOKEN "identifier" i) 430:430
(TOKEN "=") 432:432
(TOKEN "integer" 1234) 434:438
(TOKEN "+") 439:439
(TOKEN "imag number" 3456.000000j) 440:445
(NEWLINE) 446:446
(TOKEN "identifier" i) 447:447
(TOKEN "=") 449:449
(TOKEN "integer" 12) 451:452
(TOKEN "+") 453:453
(TOKEN "imag number" 0.000000j) 454:465
(NEWLINE) 466:466
(TOKEN "identifier" i) 467:467
(TOKEN "=") 469:469
(TOKEN "real" 1000.120000) 471:478
(TOKEN "+") 479:479
(TOKEN "imag number" 0.000100j) 480:486
(NEWLINE) 487:487
(NEWLINE) 488:488
(TOKEN "identifier" float) 489:493
(TOKEN "(") 494:494
(TOKEN "real" inf) 495:499
(TOKEN ")") 500:500
(NEWLINE) 501:501
(TOKEN "identifier" x) 502:502
(TOKEN "=") 504:504
(TOKEN "real" inf) 506:510
(NEWLINE) 511:511
(TOKEN "identifier" complex) 512:518
(TOKEN "(") 519:519
(TOKEN "real" 140000000000000005528749527191103381749434659621913717006437476935335702571970573026452444307820696028601680522881043700541120085625386611604505789664591785922123926873769078699987831481595335695749181141356976015104370165785971235340948584343131219149608095136586496161650337802003105866881830462290332221440.000000) 520:526
(TOKEN ",") 527:527
(TOKEN "real" 140000000000000005528749527191103381749434659621913717006437476935335702571970573026452444307820696028601680522881043700541120085625386611604505789664591785922123926873769078699987831481595335695749181141356976015104370165785971235340948584343131219149608095136586496161650337802003105866881830462290332221440.000000) 529:535
(TOKEN ")") 536:536
(NEWLINE) 537:537
(TOKEN "identifier" abs_err) 538:544
(TOKEN "=") 546:546
(TOKEN "real" 0.000000) 548:553
(NEWLINE) 554:554
(TOKEN "identifier" tiny) 555:558
(TOKEN "=") 560:560
(TOKEN "real" 0.000000) 562:567
(NEWLINE) 568:568
(TOKEN "-") 569:569
(TOKEN "real" 1.000000) 570:571
(TOKEN "+") 572:572
(TOKEN "imag number" 0.000000j) 573:579
(NEWLINE) 580:580
(TOKEN "identifier" vals) 581:584
(TOKEN "=") 586:586
(TOKEN "[") 588:588
(TOKEN "real" 0.000000) 589:594
(TOKEN ",") 595:595
(TOKEN "real" 0.000000) 597:602
(TOKEN ",") 603:603
(TOKEN "real" 0.000000) 605:610
(TOKEN "]") 611:611
(NEWLINE) 612:612
(TOKEN "identifier" z2) 613:614
(TOKEN "=") 616:616
(TOKEN "-") 618:618
(TOKEN "imag number" infj) 619:625
(NEWLINE) 626:626
(TOKEN "identifier" repr) 627:630
(TOKEN "(") 631:631
(TOKEN "real" 1000000000000000052504760255204420248704468581108159154915854115511802457988908195786371375080447864043704443832883878176942523235360430575644792184786706982848387200926575803737830233794788090059368953234970799945081119038967640880074652742780142494579258788820056842838115669472196386865459400540160.000000) 632:636
(TOKEN "*") 638:638
(TOKEN "real" 1000000000000000052504760255204420248704468581108159154915854115511802457988908195786371375080447864043704443832883878176942523235360430575644792184786706982848387200926575803737830233794788090059368953234970799945081119038967640880074652742780142494579258788820056842838115669472196386865459400540160.000000) 640:644
(TOKEN "*") 646:646
(TOKEN "integer" 0) 648:648
(TOKEN ")") 649:649
(NEWLINE) 650:650
(TOKEN "identifier" weights) 651:657
(TOKEN "=") 658:658
(TOKEN "[") 659:659
(TOKEN "real" 0.000000) 660:665
(TOKEN ",") 666:666
(TOKEN "real" 0.000000) 668:673
(TOKEN "]") 674:674
(NEWLINE) 675:675
(EOF) 676:676
